pkg/minikube/machine/start.go

```diff
@@ -202,6 +205,26 @@ func timedCreateHost(h *host.Host, api libmachine.API, t time.Duration) error {
 	}
 }
 
+// postStartValidations are validations against the host after it is created
+func postStartValidations(h *host.Host, drvName string) {
+	if !driver.IsKIC(drvName) {
+		return
+	}
+	// make sure /var isn't full, otherwise warn
+	output, err := h.RunSSHCommand("df -h /var | awk 'NR==2{print $5}'")
+	if err != nil {
+		glog.Warning("error running df -h /var: %v", err)
+	}
+	output = strings.Trim(output, "\n")
+	percentageFull, err := strconv.Atoi(output[:len(output)-1])
+	if err != nil {
+		glog.Warning("error getting % of /var that is free: %v", err)
+	}
+	if percentageFull >= 99 {
```

Comment:
I think we should also warm them at 80% but maybe with more friendly they hey you are getting close to fill up the disk.

Reply:
Cool, added that in